<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestMainWindow</class>
 <widget class="QMainWindow" name="TestMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>298</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ThermoFun Test Main Window</string>
  </property>
  <widget class="QWidget" name="centralWidget_">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>4</number>
    </property>
    <property name="topMargin">
     <number>4</number>
    </property>
    <property name="rightMargin">
     <number>4</number>
    </property>
    <property name="bottomMargin">
     <number>4</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>463</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew_ThermoFun_Window"/>
    <addaction name="separator"/>
    <addaction name="actionE_xit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionPreferences"/>
    <addaction name="separator"/>
    <addaction name="actionContents"/>
    <addaction name="separator"/>
    <addaction name="actionVersion"/>
    <addaction name="actionAuthors"/>
    <addaction name="actionLicense"/>
    <addaction name="separator"/>
    <addaction name="action_Help_About"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="action_Keep_Data_Fields_Expanded"/>
    <addaction name="action_Show_comments"/>
    <addaction name="action_Display_enums"/>
    <addaction name="action_Edit_id"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_View"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPreferences"/>
   <addaction name="actionNew_ThermoFun_Window"/>
   <addaction name="separator"/>
   <addaction name="action_Help_About"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionE_xit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="icon">
    <iconset resource="../ThermoFun.qrc">
     <normaloff>:/menu/Icons/SetKeyTemplateIcon24.png</normaloff>:/menu/Icons/SetKeyTemplateIcon24.png</iconset>
   </property>
   <property name="text">
    <string>Open &amp;Preferences Dialog...</string>
   </property>
   <property name="toolTip">
    <string>Open Preferences Dialog</string>
   </property>
  </action>
  <action name="action_Help_About">
   <property name="icon">
    <iconset resource="../ThermoFun.qrc">
     <normaloff>:/menu/Icons/QuestionMessageIcon24.png</normaloff>:/menu/Icons/QuestionMessageIcon24.png</iconset>
   </property>
   <property name="text">
    <string>Open &amp;Help Viewer ...</string>
   </property>
   <property name="iconText">
    <string>Open Help Viewer ...</string>
   </property>
   <property name="toolTip">
    <string>Open Help Viewer ...</string>
   </property>
  </action>
  <action name="actionAuthors">
   <property name="text">
    <string>Au&amp;thors ...</string>
   </property>
  </action>
  <action name="actionLicense">
   <property name="text">
    <string>&amp;License ...</string>
   </property>
  </action>
  <action name="actionContents">
   <property name="text">
    <string>Help &amp;Contents ...</string>
   </property>
  </action>
  <action name="action_Show_comments">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Display Comments to Data Fields</string>
   </property>
  </action>
  <action name="action_Display_enums">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Show Enum Names Instead of Values</string>
   </property>
  </action>
  <action name="actionVersion">
   <property name="text">
    <string>&amp;Version ...</string>
   </property>
  </action>
  <action name="action_Edit_id">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../ThermoFun.qrc">
     <normaloff>:/menu/Icons/CancelIcon24.png</normaloff>:/menu/Icons/CancelIcon24.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Edit _id of DB Documents (Records) </string>
   </property>
  </action>
  <action name="action_Keep_Data_Fields_Expanded">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Keep Data Fields Expanded</string>
   </property>
  </action>
  <action name="actionNew_ThermoFun_Window">
   <property name="icon">
    <iconset resource="../ThermoFun.qrc">
     <normaloff>:/menu/Icons/DatabaseFunctionIcon24.png</normaloff>:/menu/Icons/DatabaseFunctionIcon24.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New ThermoFun Window ...</string>
   </property>
   <property name="toolTip">
    <string>New ThermoFun Data Editor Window</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../ThermoFun.qrc"/>
 </resources>
 <connections/>
</ui>
